packages/fast-components-react-base/src/button/button.props.ts

```diff
@@ -1,24 +1,41 @@
 import * as React from "react";
 import { IButtonClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
 
-/**
- * Button HTML tags
- */
-export enum ButtonHTMLTags {
-    a = "a",
-    button = "button"
-}
-
 export interface IButtonHandledProps {
     /**
-     * The HTML tag (defaults to ButtonHTMLTags.button)
+     * The button content
      */
-    tag?: ButtonHTMLTags;
+    children?: React.ReactNode | React.ReactNode[];
 
     /**
-     * The button content
+     * The disabled state
      */
-    children?: React.ReactNode | React.ReactNode[];
+    disabled?: boolean;
+
+    /**
+     * The destination address (turns button to anchor)
+     */
+    href?: string;
+
+    /**
```

Comment:
Should these live at the base level, or should these be added in a "Fancy Button" instance in MSFT?

Reply:
Yes - updating